lib/matplotlib/legend.py

```diff
@@ -111,7 +111,7 @@ def _update_bbox_to_anchor(self, loc_in_canvas):
 
 
 _legend_kw_doc = '''
-loc : int or string or pair of floats, default: 'upper right'
+loc : int or string or pair of floats, default: :rc:`legend.loc`
```

Comment:
Can you please also state the default rc value like this:

``default: :rc:`legend.loc` = 'best'``

Reply:
Ah, didn't read your text completely.

We would like to add the rc default values in the future (there's a pending PR #10714 for this)
See section **rcParams** in https://10714-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/devel/documenting_mpl.html?highlight=documenting#writing-docstrings.

It's true that this is a bit more involved in the case of legend. I think your propoal

~~~
default: :rc:`legend.loc` ('best' for axes, 'upper right' for figures)
~~~

is reasonable.

I wouldn't want a default issueing a warning that it will be changed. This is rather confusing. Of course, the best way, would be to have 'best' also for figures. Since we don't have that for now, the different defaults make sense.

Reply:
Great thanks for letting me know about that, including the default param value is a welcome change, looking forward!

I wasn't sure which line break strategy to use for the parentheses here. I went with the same approach as in [`figure.py` lines 695-696](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/figure.py#L695-L696), as this was the only other occurrence of a line breaking default I could find. It does make the line quite long compared to other lines in the docstring, so I can add a line break instead if you prefer. I just need advice on how much I should indent the line after the break.